ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -544,8 +563,13 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         tap(isBusy => this.componentBusy.emit(isBusy))
     );
 
-    const emitValidStatusAction$ = combineLatest([this.formErrorMessages$, this.addressErrors$]).pipe(
-        map(([formErrors, addressErrors]) => !formErrors.length && !addressErrors.length),
+    const activitityRequirementsMet$: Observable<boolean> = currentAddress$.pipe(
+        map(currentAddress => this.meetsActivityRequirements(currentAddress))
+    );
+
+    const emitValidStatusAction$ = combineLatest([this.formErrorMessages$, this.verifyFieldErrors$, activitityRequirementsMet$]).pipe(
+        map(([formErrors, addressErrors, reqsMet]) =>
+            !formErrors.length && !addressErrors.length && reqsMet),
         distinctUntilChanged(),
         tap(status => this.validStatusChanged.emit(status))
     );
```

Comment:
We emit the validation status of the component. In addition to having form errors, we now take a look as well to the new constraints coming from server (particularly isPhoneRequired)